### PR TITLE
Update Localizable.strings - translated some of the lines from English to Ukrainian

### PR DESCRIPTION
**Pull request by rbolgaryn**
### First time contributor checklist

- [x] I have read the [README](https://github.com/WhisperSystems/Signal-iOS/blob/master/README.md) and [CONTRIBUTING](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md) documents
- [ ] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I'm following the [code, UI and style conventions](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md#code-conventions)
- [x] My commits are rebased on the latest master branch
- [x] My commits are in nice logical chunks
- [x] My contribution is fully baked and is ready to be merged as is
- [ ] I have tested my contribution on these devices:
 * iDevice A, iOS X.Y.Z
 * iDevice B, iOS Z.Y

- - - - - - - - - -

### CODE REVIEW

**Comment by rbolgaryn** [comment]
Reopened because I could not find a way to edit Ukrainian strings through Transifex (somehow the dialog does not let you choose Ukrainian).

File: Signal/translations/uk.lproj/Localizable.strings [hunk 1]
```diff
@@ -3425,10 +3425,10 @@
 "PIN_CREATION_CREATE_NUMERIC" = "Створити цифровий PIN-код";
 
 /* Action of the 'pin disable' action sheet. */
-"PIN_CREATION_DISABLE_CONFIRMATION_ACTION" = "Disable PIN";
+"PIN_CREATION_DISABLE_CONFIRMATION_ACTION" = "Вимкнити PIN";
```
Comment (Sky) [inline Signal/translations/uk.lproj/Localizable.strings:3429]: Вимкнути?

Comment (rbolgaryn) [reply Signal/translations/uk.lproj/Localizable.strings:3429]: fixed

**Comment by Imperiopolis-Signal** [comment]
Translations must be edited in transifex.

**Pull request closed without merging**